File: sampleActivity/app/src/main/java/com/example/sampleactivity/MainActivity.java
package com.example.sampleactivity;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void findingSimpleInterest(View view) {
        TextView result=(TextView)findViewById(R.id.textView);
        result.setText("  the result is: ");
    }
}

File: HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.kt
package com.example.helloworld

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
    fun sendMessage(View:view){
        Intent intent=new Intent(this ,MessageActivity.class)
        startActivity(intent)
    }
}
